From 0942d9245ed1a7de573e3af17deac2332a52b58a Mon Sep 17 00:00:00 2001
From: Michal Zidek <mzidek@redhat.com>
Date: Mon, 24 Nov 2014 19:10:01 +0100
Subject: [PATCH 17/26] test: Wrong parameter type in sss_parse_name_check
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This caused aritmetic overflow when SSSD specific error
codes where used.

Reviewed-by: Lukáš Slebodník <lslebodn@redhat.com>
---
 src/tests/cmocka/test_fqnames.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/cmocka/test_fqnames.c b/src/tests/cmocka/test_fqnames.c
index 71429c8773ef199c72163837d4b313660cf813c2..de208437d3d11429ebb4fd92ac6b1469564d9174 100644
--- a/src/tests/cmocka/test_fqnames.c
+++ b/src/tests/cmocka/test_fqnames.c
@@ -326,7 +326,7 @@ void parse_name_test_teardown(void **state)
 
 void sss_parse_name_check(struct parse_name_test_ctx *test_ctx,
                           const char *input_name,
-                          const char exp_ret,
+                          const int exp_ret,
                           const char *exp_name,
                           const char *exp_domain)
 {
-- 
2.1.0

